drivers/clock_control/clock_stm32l0_l1.c

```diff
@@ -26,29 +25,44 @@
 #define pll_div(v) z_pll_div(v)
 
 /**
- * @brief Set up pll configuration
+ * @brief Return PLL source
  */
-int config_pll_sysclock(void)
+__unused
+static uint32_t get_pll_source(void)
 {
-	uint32_t pll_source, pll_mul, pll_div;
-
-	pll_mul = pll_mul(STM32_PLL_MULTIPLIER);
-	pll_div = pll_div(STM32_PLL_DIVISOR);
-
 	/* Configure PLL source */
 	if (IS_ENABLED(STM32_PLL_SRC_HSI)) {
-		pll_source = LL_RCC_PLLSOURCE_HSI;
+		return LL_RCC_PLLSOURCE_HSI;
 	} else if (IS_ENABLED(STM32_PLL_SRC_HSE)) {
-		pll_source = LL_RCC_PLLSOURCE_HSE;
-	} else {
-		return -ENOTSUP;
+		return LL_RCC_PLLSOURCE_HSE;
 	}
 
-	LL_RCC_PLL_ConfigDomain_SYS(pll_source, pll_mul, pll_div);
-
+	__ASSERT(0, "Invalid source");
```

Comment:
Minor.

Some `__ASSERT(0, "Invalid source");` use uppercase for `Invalid`, while others (i.e.  `clock_stm32g4.c` line `40`) use lowercase `invalid source`.

Would be better to use single format approach.

Reply:
Done